sled-agent/src/sim/config.rs

```diff
@@ -60,12 +70,55 @@ pub struct Config {
     pub nexus_address: SocketAddr,
     /// configuration for the sled agent dropshot server
     pub dropshot: ConfigDropshot,
-    /// configuration for the sled agent debug log
-    pub log: ConfigLogging,
```

Comment:
This was not used in almost any callers, so a bunch of them faked up a value.  I separated it to a separate argument to the one function where it was used (`run_standalone_server`) and removed the faked up values.